src/locales/en_US.js

```diff
@@ -677,10 +677,11 @@ const strings = {
   fio_address_details_screen_alert_button: 'View Details',
   fio_address_details_screen_registered: 'Registered',
   fio_address_details_screen_expires: 'Expires',
-  fio_address_details_screen_bundles: 'Remaining bundles',
+  fio_address_details_screen_bundles: 'Remaining bundled transactions',
   fio_address_details_screen_manage_account_settings: 'Address Settings',
-  fio_address_details_bundles_out_soon: "Your FIO address's bundled transactions is almost spent. Would you like to purchase more bundles?",
-  fio_address_details_no_bundles: 'Your FIO address has no bundled transactions. Would you like to purchase more bundles?',
+  fio_address_details_bundles_out_soon:
+    'You have spent almost all the bundled transactions of your FIO address. Would you like to purchase more bundled transaction?',
+  fio_address_details_no_bundles: 'Your FIO address has no bundled transactions. Would you like to purchase more bundled transactions?',
```

Comment:
Please change all variables that have bundles to bundledTxs or bundled_txs bundles is not a proper name